backend/controllers/itemController.js

```diff
@@ -162,11 +162,7 @@ const removeSingularItem = asyncHandler(async (req, res) => {
 // @access Private
 const createSingleItem = asyncHandler(async (req, res) => {
     const { classCode, itemName } = req.body;
-    let item = await Item.findOne({ itemName });
-    if (!item) {
-        return res.status(404).json({ error: "Item not found in the equipment checkout center." });
```

Comment:
This peace of code cannot be remove, since class code need to choose from all item, if this item is not in inventory center, then it cannot add to the classcode

Reply:
other things looks good